documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -6,14 +6,18 @@
 
 * Container Scan. Providing component risk detail analysis for each layer of a container image, (including non-Linux, non-Docker images). Please see [Container Scan ](runningdetect/containerscanning.md) for details.
 	<note type="restriction">Your [blackduck_product_name] server must have [blackduck_product_name] Secure Container (BDSC) licensed and enabled.</note>
+* Support for Dart is now extended to Dart 3.1.2 and Flutter 3.13.4.
+* Documentation for [CPAN Package Manager](packagemgrs/cpan.md) and [BitBucket Integration](integrations/bitbucket/bitbucketintegration.md) has been added.
 
 ### Changed features
 
 * For Signature Scans, the directory exclusion argument is no longer passed by default, thus allowing the examination of directory content for matches. Directories can still be excluded from matching by using the '–detect.blackduck.signature.scanner.arguments' property. Please see the [Signature Scanner](properties/configuration/signature-scanner.md#signature-scanner-arguments) property documentation for information on how to specify directories for exclusion.
-* Support for Dart is now extended to Dart 3.1.2 and Flutter 3.13.4.
-* When [blackduck_product_name] is busy, [solution_name] will now wait the number of seconds specified by [blackduck_product_name] before attempting to retry scan creation.
+* When [blackduck_product_name] version 2023.10.0 or later is busy and includes a retry-after value greater than 0 in the header, [solution_name] will now wait the number of seconds specified by [blackduck_product_name] before attempting to retry scan creation. 
+	* [solution_name] 9.1.0 will not retry scan creation with versions of [blackduck_product_name] prior to 2023.10.0
 
 ### Resolved issues
+* (IDETECT-3843) Additional information is now provided when [solution_name] fails to update and [solution_name] is internally hosted.
 * (IDETECT-4056) Resolved an issue where no components were reported by CPAN detector.
   If the cpan command has not been previously configured and run on the system, [solution_name] instructs CPAN to accept default configurations.
-* (IDETECT-3843) Additional information is now provided when [solution_name] fails to update and [solution_name] is internally hosted.
\ No newline at end of file
+* (IDETECT-4005) Resolved an issue where the location is not identified for a Maven component version when defined as a property.
+* (IDETECT-4074) Resolved Widows project creation issue where leading or trailing spaces in a project name was creating a project with default name instead of the desired name.
```

Comment:
Not sure if IDETECT-4074 needs mention. No Detect changes were made except doc clarification. Also, this was an internally found ticket.

Reply:
Karolyn makes a good point. If we do keep it there is a typo in Windows

Reply:
Doh! The typo alone is reason enough.... I will remove it.